assets/Templates/stratcards/codex1/39364D824F5CB1EFAAE8928976925102.json

```diff
@@ -22,10 +22,10 @@
 		"G": 0,
 		"B": 0
 	},
-	"Flippable": false,
+	"Flippable": true,
 	"AutoStraighten": false,
 	"ShouldSnap": true,
-	"ScriptName": "",
+	"ScriptName": "gameobjects/strategy-card.js",

```

Comment:
shouldn't have a script anymore

the Metadata here is "tile.strategy:codex.ordinian/diplomacy", keeping the ":base" version should someone choose to use it.